build.gradle

```diff
@@ -15,39 +11,6 @@ plugins {
 
 ext.native_targets_enabled = !project.hasProperty("disable_native_targets")
 
-allprojects {
-    // the only place where HostManager could be instantiated
-    project.ext.hostManager = new HostManager()
-
-    logger.info("Using Kotlin compiler ${libs.versions.kotlin} for project ${project.name}")
-
-
-    def deployVersion = project.findProperty("DeployVersion")
-    if (deployVersion != null) project.version = deployVersion
-
-    // this fixes "org.gradle.jvm.version" in Gradle metadata
-    plugins.withType(JavaPlugin).configureEach {
-        java {
-            toolchain {
-                languageVersion.set(JavaLanguageVersion.of(8))
-            }
-        }
-    }
```

Comment:
Just to double check, is the language version config still present in the new plugins?

Reply:
I didn't remove it, just changed to:
```
kotlin {
    jvmToolchain(8)
}
```
I've checked it by comparing published artifacts and artifacts before changes and after are equal. Or should I check it other way?

Reply:
Great, thanks for checking. I just wanted to confirm, no further action needed 👍